larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -448,11 +448,11 @@ public void printHtmlForm(List<String> scenariosRootDirectories, List<String> sc
 		writeHtml("<span style=\"float: left; font-size: 10pt; width: 0px\">&nbsp; &nbsp; &nbsp;</span>", false);
 		writeHtml("<table style=\"float:left;height:50px\">", false);
 		writeHtml(TR_STARTING_TAG, false);
-		writeHtml("<td>Multi threaded (beta)</td>", false);
+		writeHtml("<td>Multi Threaded (experimental)</td>", false);
 		writeHtml(TR_CLOSING_TAG, false);
 		writeHtml(TR_STARTING_TAG, false);
 		writeHtml(TD_STARTING_TAG, false);
-		writeHtml("<input type=\"checkbox\" name=\"multithreaded\" value=\"true\"", false);
+		writeHtml("<input type=\"checkbox\" name=\"multithreaded\"", false);
```

Comment:
This is not consistent with other checkboxes, which all send the value `"true"` when checked.
Writing value `true` doesn't mean the checkbox is checked, that is done on line 457.

So I suggest reverting this and like 182, to be consistent with all other checkboxes in the file.

See for instance:
https://stackoverflow.com/questions/11424037/do-checkbox-inputs-only-post-data-if-theyre-checked

Reply:
What does this achieve?

Reply:
Consistency with rest of the code, that writes checkboxes with `value="true"` also when not checked.